### DISCUSSION TITLE
why are some files green?

### DISCUSSION THREAD

**Discussion by raylu** [eza-community/eza#1034] [Q&A, Howto, Help] [answered]
I've poked around a bit and read https://the.exa.website/docs/colour-themes, but I guess I don't understand what the default, out-of-the-box configuration is

when running `eza` without any flags, `LS_COLORS`, or `EZA_COLORS` on the eza repo, the `deb.asc` file shows up green. why is that (and how can I stop non-executable files from showing up that color)?

**Comment by gierens** [comment, marked as answer]
Hi @raylu 

the reason is that green is the default color for crypto-related files, see: https://github.com/eza-community/eza/blob/d414d4bf50e4ed47b5dd82b4f347633782aadf27/src/theme/default_theme.rs#L102

You can change this using `EZA_COLORS` via the `cr` code, for example:
![Screenshot from 2024-07-08 09-22-30](https://github.com/eza-community/eza/assets/49617392/d795db96-cca0-4ab2-99f2-f6d3c1f36b54)

In addition we created a config file PR awaiting its final review and merge. It would not only allow to configure the style (color, icon, ...) of file types but on a file name and file extension basis, see https://github.com/eza-community/eza/pull/840

Hope this answers your question.

**Reply by raylu** [reply]
ah, thanks! together with https://github.com/eza-community/eza/blob/main/man/eza_colors.5.md, I was able to come up with
```sh
export EZA_COLORS=bu=0:cr=0
```
which disables "build" and "crypto" files from being yellow, leaving just "source" files
